[CI] StreamsYamlTestSuiteIT test {yaml=streams/logs/10_basic/Basic toggle of logs state enable to disable and back} failing

**Build Scans:**
- [elasticsearch-periodic #8847 / release-tests](https://gradle-enterprise.elastic.co/s/u6c7cper6gnqs)
- [elasticsearch-pull-request #75968 / part-1](https://gradle-enterprise.elastic.co/s/imgp2zkh24mis)

**Reproduction Line:**
```
./gradlew ":modules:streams:yamlRestTest" --tests "org.elasticsearch.streams.StreamsYamlTestSuiteIT.test {yaml=streams/logs/10_basic/Basic toggle of logs state enable to disable and back}" -Dtests.seed=AF868F4A49FD3A2B -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dlicense.key=x-pack/license-tools/src/test/resources/public.key -Dtests.locale=zh-Hans-SG -Dtests.timezone=America/Merida -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.streams.StreamsYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(test%20%7Byaml%3Dstreams%2Flogs%2F10_basic%2FBasic%20toggle%20of%20logs%20state%20enable%20to%20disable%20and%20back%7D),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [streams/logs/10_basic:3]: expected [2xx] status code but api [streams.logs_enable] returned [400 Bad Request] [{"error":"no handler found for uri [/_streams/logs/_enable?error_trace=true] and method [POST]"}]
```

**Issue Reasons:**
- [main] 2 failures in test test {yaml=streams/logs/10_basic/Basic toggle of logs state enable to disable and back} (4.9% fail rate in 41 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/modules/streams/src/yamlRestTest/java/org/elasticsearch/streams/StreamsYamlTestSuiteIT.java
+++ b/modules/streams/src/yamlRestTest/java/org/elasticsearch/streams/StreamsYamlTestSuiteIT.java
@@ -13,6 +13,7 @@ import com.carrotsearch.randomizedtesting.annotations.Name;
 import com.carrotsearch.randomizedtesting.annotations.ParametersFactory;
 
 import org.elasticsearch.test.cluster.ElasticsearchCluster;
+import org.elasticsearch.test.cluster.FeatureFlag;
 import org.elasticsearch.test.rest.yaml.ClientYamlTestCandidate;
 import org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase;
 import org.junit.ClassRule;
@@ -28,10 +29,11 @@ public class StreamsYamlTestSuiteIT extends ESClientYamlSuiteTestCase {
     }
 
     @ClassRule
-    public static ElasticsearchCluster cluster = ElasticsearchCluster.local().module("streams").build();
+    public static ElasticsearchCluster cluster = ElasticsearchCluster.local().module("streams").feature(FeatureFlag.LOGS_STREAM).build();
 
     @Override
     protected String getTestRestCluster() {
         return cluster.getHttpAddresses();
     }
+
 }

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -568,12 +568,6 @@ tests:
 - class: org.elasticsearch.search.query.RescoreKnnVectorQueryIT
   method: testKnnSearchRescore
   issue: https://github.com/elastic/elasticsearch/issues/129713
-- class: org.elasticsearch.streams.StreamsYamlTestSuiteIT
-  method: test {yaml=streams/logs/10_basic/Basic toggle of logs state enable to disable and back}
-  issue: https://github.com/elastic/elasticsearch/issues/129733
-- class: org.elasticsearch.streams.StreamsYamlTestSuiteIT
-  method: test {yaml=streams/logs/10_basic/Check for repeated toggle to same state}
-  issue: https://github.com/elastic/elasticsearch/issues/129735
 - class: org.elasticsearch.snapshots.GetSnapshotsIT
   method: testFilterByState
   issue: https://github.com/elastic/elasticsearch/issues/129740

